userguide/jails.rst

```diff
@@ -274,72 +255,71 @@ settings, click :guilabel:`Next` to proceed to the
+   |                       |           | :guilabel:`allow_mount_devfs` permissions are enabled and :guilabel:`enforce_statfs` is set to a value lower than   |
+   |                       |           | *2*.                                                                                                                |
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
-   | exec_start            | string    | Commands to run in the prison environment when a jail is created. Example: :samp:`sh /etc/rc`. See                  |
+   | exec.start            | string    | Commands to run in the jail environment when a jail is created. Example: :samp:`sh /etc/rc`. See                    |
    |                       |           | `jail(8) <https://www.freebsd.org/cgi/man.cgi?query=jail&manpath=FreeBSD+11.1-RELEASE+and+Ports>`__                 |
    |                       |           | for more details.                                                                                                   |
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
-   | exec_stop             | string    | Commands to run in the prison environment before a jail is removed and after any :guilabel:`exec_prestop`           |
-   |                       |           | commands have completed. Example: :samp:`sh /etc/rc.shutdown`.                                                      |
+   | exec.stop             | string    | Commands to run in the jail environment before a jail is removed and after any :guilabel:`exec_prestop` commands    |
+   |                       |           | are complete. Example: :samp:`sh /etc/rc.shutdown`.                                                                 |
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
-   | exec_prestart         | string    | List any commands to run in the system environment before a jail is started.                                        |
+   | exec_prestart         | string    | Commands to run in the system environment before a jail is started.                                                 |
```

Comment:
We have rewritten this before, I remember it.
Specifically, "in the system environment" has no meaning.  Is it in the jail?  In the host?